microcosm_pubsub/dispatcher.py

```diff
@@ -1,6 +1,31 @@
 """
 Process batches of messages.
 
+Flow diagram:
+
++-----------------------+                        +-------------+                                            +----------+
+|                       |                        |             |                                            |          |
+|                       |     1. Get messages    |             |                                            |          |
```

Comment:
So the `SQSMessageDispatcher` gets messages from `SQSConsumer` or vice-versa? I'm asking because arrows point to both nodes.

Reply:
The dispatcher gets messages from the SQS consumer.

I'm guessing "propagate result" means `ack`/`nack`? Then that's the second part of that bidirectional flow.

I think making that arrow go both ways is a bit confusing however. I think if you removed that (indicating that the dispatcher takes messages from the consumer, handles/acks them), then "get messages" could be assumed to be the next part of the dispatchers lifecycle.

Reply:
Updated arrow direction.
As for "propagate result", I moved it to have the explanation directly underneath.